python/py-maxminddb/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                maxminddb
+version             1.5.2
+revision            0
+categories-append   python
+platforms           darwin
+license             Apache Software License (Apache License, Version 2.0)
+
+python.versions     27 35 36 37 38
+
+maintainers         {@gstaniak gmail.com:gstaniak}
+
+description         Reader for the MaxMind DB format
+
+long_description    This is a Python module for reading MaxMind DB files. The module \
+                    includes both a pure Python reader and an optional C extension.\
+                    \
+                    MaxMind DB is a binary file format that stores data indexed by \
+                    IP address subnets (IPv4 or IPv6).
+
+homepage            https://www.maxmind.com/en/home
+
+checksums           rmd160 caaa546c1daae6781c45a17be906a21a3b4ff654 \
+                    sha256 d0ce131d901eb11669996b49a59f410efd3da2c6dbe2c0094fe2fef8d85b6336 \
+                    size 274594
+
+depends_lib-append  port:libmaxminddb
+depends_lib-append  port:py${python.version}-setuptools
+
+livecheck.type      none
```

Comment:
These 3 lines should all be enclosed in a if `{${name} ne ${subport}} {` block because they are not correct for the stub port.